README-build-from-source.md

```diff
@@ -1,194 +0,0 @@
-This document describes how to build ATF, UEFI, Linux kernel and Android for HiKey960 from source.
```

Comment:
Hi, John

I have two questions:
1. UEFI instructions are also required by people who want to have a preview. So I want this information be kept somewhere.
2. Why do you remove linux kernel and Android build instruction?

How about I create a different folder, saying "build-from-source", and put related files "mkdtimg", "mkbootimg", and "README-build-from-source.txt" into it?